src/main/java/org/alliancegenome/curation_api/view/View.java

```diff
@@ -7,9 +7,10 @@ public static class FieldsOnly { }
 	public static class FieldsAndLists extends FieldsOnly { }
 	public static class ConditionRelationUpdateView extends FieldsOnly { }
 
-	public static class VocabularyTermView extends FieldsAndLists { }
+	public static class VocabularyTermView extends FieldsOnly { }
```

Comment:
Would be curious all the places that use the VocabularyTermView and make sure this wouldn't cause any issues.

Reply:
Don't think I intended to switch that - will switch it back